From 8136650b1047d8204df19a8bd37285d9a21421fe Mon Sep 17 00:00:00 2001
From: "Le.Ma" <Le.Ma@amd.com>
Date: Thu, 12 Oct 2017 13:29:25 +0800
Subject: [PATCH 1840/4131] drm/amdgpu: [RHEL 6] Use DEFINE_IDA2 instead on
 rhel6

The idr/ida interfaces was redirected to own idr/ida in drm_backport.h on rhel6

Change-Id: I9df12451187adc779779173acbd3666931fa0090
Signed-off-by: Le.Ma <Le.Ma@amd.com>
Reviewed-by: Junwei Zhang <Jerry.Zhang@amd.com>
---
 drivers/gpu/drm/amd/amdgpu/amdgpu_vm.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/drivers/gpu/drm/amd/amdgpu/amdgpu_vm.c b/drivers/gpu/drm/amd/amdgpu/amdgpu_vm.c
index ff80728..b5cb6ca 100644
--- a/drivers/gpu/drm/amd/amdgpu/amdgpu_vm.c
+++ b/drivers/gpu/drm/amd/amdgpu/amdgpu_vm.c
@@ -48,7 +48,11 @@
  * space. Therefore PASIDs are allocated using a global IDA. VMs are
  * looked up from the PASID per amdgpu_device.
  */
+#if !defined(OS_NAME_RHEL_6)
 static DEFINE_IDA(amdgpu_vm_pasid_ida);
+#else
+static DEFINE_IDA2(amdgpu_vm_pasid_ida);
+#endif
 
 /**
  * amdgpu_vm_alloc_pasid - Allocate a PASID
-- 
2.7.4

